corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,6 +450,69 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
+        columnStyle: function () {
+            let self = this;
+            let mapColumn = "";
+            if (self.showMap && !self.hasNoItems) {
+                mapColumn = "[map] 300px";
+            }
+            return "display: grid;grid-template-columns: [tiles] auto " + mapColumn
```

Comment:
What's the `[tiles]` / `[map]` syntax? I'm not familiar with that.

Reply:
That is just naming the columns. I think you can use that as references in other CSS properties but I just added them to make it easier to understand.